### ISSUE TITLE
Finish CAIN sheet aesthetics

### ISSUE THREAD

**Issue by benjo121ben** [diabeatz96/cain#74]
Update the ID section and do some other minor tweaks to the UI

**Comment by Orphansmith** [comment]
Commenting to follow and ask what exactly this request is for?

I think the system works fine, but the sheet CSS and architecture could use some work.

**Comment by diabeatz96** [comment]
We can open a new issue for the things that you think need fixing, I believe this was when the sheet was in preliminary stages and we tweaked a lot of the CSS to just be less inconsistent/look nicer.

**Comment by benjo121ben** [comment]
Yes, pretty sure it was something along those lines. Not sure if the changes I was working on were ever finished and merged.
Sorry for the lackluster description of this issue. I'd recommend just closing it
